erpnext/stock/stock_balance.py

```diff
@@ -94,51 +98,85 @@ def get_balance_qty_from_sle(item_code, warehouse):
 
 
 def get_reserved_qty(item_code, warehouse):
```

Comment:
This function looks very complex and difficult to read, could you split the code into multiple methods?

Reply:
@rohitwaghchaure Honestly, I don't even fully understand the query. I just reverted it back to the previous query and used f string to inject the condition.